webroot/js/toolbar.js

```diff
@@ -94,15 +94,15 @@ if (elem) {
 
   // Bind on ready callback.
   if (doc.addEventListener) {
-    doc.addEventListener('DOMContentLoaded', onReady, false);
-    doc.addEventListener('DOMContentLoaded', proxyAjaxOpen, false);
-    doc.addEventListener('DOMContentLoaded', proxyAjaxSend, false);
+    if (!window.__debugKitListenersApplied) {
```

Comment:
I decided to also wrap the `DOMContentLoaded` listeners to improve performance

For further information see [my previous comment](https://github.com/cakephp/debug_kit/pull/664/commits/c05cfbf23c128c152425d100b7cad0c08964d28a#r270721674)